android/res/values/strings.xml

```diff
@@ -92,6 +92,7 @@
     <string name="use_tpb">Use TPB</string>
     <string name="use_zooqle">Use Zooqle.com</string>
     <string name="use_yify">Use Yify</string>
+    <string name="use_one337x">Use 1337x</string>
```

Comment:
It'd be great if you'd add this string to all the other languages, or at least Spanish, French, German, Portuguese, Italian, Hindi